Froggie/Source/Froggie.Domain/Users/UserRegisterService.cs

```diff
@@ -48,6 +53,9 @@ public async ValueTask<User> RegisterAsync(string emailValue, string nameValue,
         var password = new Password(passwordValue);
         await addUserCommand.AddAsync(user, password);
 
+        // TODO: Add to a new RegisterResult.
```

Comment:
Right now I'm returning a `LogInResponse` when registering a new user but I think registering is different enough that it should have its own return type.